src/web/browser/coreVitals/coreVitals.ts

```diff
@@ -52,20 +63,16 @@ export const coreVitals = (): void => {
 				break;
 		}
 
-		const endpoint =
-			window.location.hostname === 'm.code.dev-theguardian.com' ||
-			window.location.hostname === 'localhost' ||
-			window.location.hostname === 'preview.gutools.co.uk'
-				? 'http://performance-events.code.dev-guardianapis.com/core-web-vitals'
-				: 'https://performance-events.guardianapis.com/core-web-vitals';
-
-		// If CLS has been calculated
-		if (jsonData.cls !== null) {
-			// Set page view and browser ID
-			if (window.guardian && window.guardian.ophan) {
-				jsonData.page_view_id = window.guardian.ophan.pageViewId;
-				jsonData.browser_id = window.guardian.config.ophan.browserId;
+		// Set page view and browser ID
+		if (window.guardian.config.ophan.browserId && window.guardian.ophan) {
+			jsonData.page_view_id = window.guardian.ophan.pageViewId;
+			jsonData.browser_id = window.guardian.config.ophan.browserId;
+		}
 
+		// Check if POST has been sent
+		if (!hasBeenSent) {
+			// If CLS has been calculated
+			if (jsonData.cls !== null) {
```

Comment:
You can group these two conditional I presume?

Reply:
Sure, I just did it as an initial readability thing